packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -265,7 +265,21 @@ function createProjectionAndParams({
                 ...(context.auth ? ["auth: $auth"] : []),
                 ...(context.cypherParams ? ["cypherParams: $cypherParams"] : []),
             ];
-            const apocParams = Object.entries(field.args).reduce(
+
+            // Null default argument values are not passed into the resolve tree therefore these are not being passed to
+            // `apocParams` below causing a runtime error when executing.
+            const nullArgumentValues = cypherField.arguments.reduce(
+                (r, argument) => ({
+                    ...r,
+                    ...{ [argument.name.value]: null },
+                }),
+                {}
+            );
+
+            // Overwrite null arguments with those defined in resolve tree
+            const argumentsWithNullValues = { ...nullArgumentValues, ...field.args };
+
+            const apocParams = Object.entries(argumentsWithNullValues).reduce(
```

Comment:
Looks to me that this one still needs moving down into `Object.entries`?

```suggestion
            const apocParams = Object.entries({ ...nullArgumentValues, ...field.args }).reduce(
```